### PR TITLE
Update changelog.txt

### PR DESCRIPTION
**Pull request by trwalke**
Changes for 4.9.0 release

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,3 +1,12 @@
+4.9.0
+============
+New Features: ** Added support for Android Broker to MSAL.NET ** MSAL.NET will now be able to take advantage of the brokered authentication scenarios using the microsoft Authenticator and the Intune Company Portal. [Issue](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1402)
```
Comment (jennyf19) [inline changelog.txt]: >m [](start = 160, length = 1)

capitalize "m"

File: changelog.txt [hunk 2]
```diff
@@ -1,3 +1,12 @@
...
+
+**Added client capabilities support to MSAL.NET** [Issue](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1545)
+
+Bug Fixes:
+**Wrong Authority created in CreateAuthorityForRequest** MSAL.NET now properly creates B2C authorities when using the CreateAuthorityForRequest() api. [Issue](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1606)
```
Comment (pmaytak) [inline changelog.txt]: 'api' to 'API'?

**Comment by pmaytak** [comment]
Sidenote: Could this file perhaps be updated (in the future) to an .md to aid in readability?

File: changelog.txt [hunk 3]
```diff
@@ -1,3 +1,12 @@
+4.9.0
+============
+New Features: ** Added support for Android Broker to MSAL.NET ** MSAL.NET will now be able to take advantage of the brokered authentication scenarios using the Microsoft Authenticator and the Intune Company Portal. [Issue](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1402)
```
Comment (henrik-me) [inline changelog.txt]: >New Features:  [](start = 0, length = 14)

how about a link to the wiki page describing how to use it?

File: changelog.txt (hunk 2 above)
[the remark this answers is not in the document]
Comment (henrik-me) [reply changelog.txt]: I believe the fix was to use the authority from acquireToken call if had not been specified on the application

---
In reply to: [388631986](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1682#discussion_r388631986) [](ancestors = 388631986)

**Comment by henrik-me** [comment]
@trwalke ?

**Pull request merged**